### PR TITLE
 Adding tooltip to settings and menu

### PR DESCRIPTION
**Pull request by Kledis98**

### CODE REVIEW

File: src/app/header/header.component.html [hunk 1]
```diff
@@ -10,6 +10,7 @@
                     'header-button-dark-mode': darkModeService.isDarkMode
                 }"
                 type="button"
+                title="hap menunë"
```
Comment (AndiBraimllari) [inline src/app/header/header.component.html:13]: A change to "hap menynë" has been asked from linguistics, let's swap it quickly.

**Review by AndiBraimllari — DISMISSED** [review]

File: src/app/header/header.component.html (hunk 1 above)
Comment (Kledis98) [reply src/app/header/header.component.html:13]: Sure !

Comment (AndiBraimllari) [reply src/app/header/header.component.html:13]: Feel free to merge after changing it, resolving conversation.

**Comment by AndiBraimllari** [comment]
Now that I think of it, perhaps we can add one to the "penda" writing saying "faqja kryesore", what do you think? @Kledis98

**Comment by Kledis98** [comment]
Yeah, we can add it too.

**Comment by Kledis98** [comment]
Would be informative.

**Pull request merged**